lib/resty/core/phase.lua

```diff
@@ -0,0 +1,58 @@
+local ffi = require 'ffi'
+local C = ffi.C
+local base = require "resty.core.base"
+
+ffi.cdef[[
+int ngx_http_lua_ffi_get_phase(ngx_http_request_t *r, char **err)
+]]
+
+local _M = {
+    _VERSION = base.version
+}
+
+local errmsg = base.get_errmsg_ptr()
+
```

Comment:
I'm afraid we use two blank lines to separate different kinds of global constructs in this library consistently.